src/main/java/hwr/oop/application/Database.java

```diff
@@ -1,34 +1,209 @@
 package hwr.oop.application;
+
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.sql.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 
-public class Database {
 
+public class Database {
+    DateTimeFormatter formatToString = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
     public Connection connect() {
-        Connection conn = null;
+        Connection connectionToDatabase = null;
         try {
-            conn = DriverManager.getConnection(System.getenv("DATABASE_FILE_URL"));
+            connectionToDatabase = DriverManager.getConnection(System.getenv("DATABASE_FILE_URL"));
         } catch (SQLException e) {
             System.out.println(e.getMessage());
         }
-        return conn;
+        return connectionToDatabase;
+    }
+
+    public static void initTables() {
+        try (Connection connectionToDatabase = DriverManager.getConnection(System.getenv("DATABASE_FILE_URL"));
+             Statement preparedStatement = connectionToDatabase.createStatement()) {
+            String sqlFile = new String(Files.readAllBytes(Paths.get(System.getenv("SQL_FILE_URL"))));
+            preparedStatement.executeUpdate(sqlFile);
+        } catch (SQLException | IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void insertIntoProject(String title, String description, LocalDateTime dateTimeCreated, LocalDateTime dateTimeDeadline, int ownerId) {
```

Comment:
```suggestion
    public void insertIntoProject(Project project) {
```
Viele Parameter machen Code unübersichtlich. Da das alles Attribute von dem Object `Project` sind, kann man auch zu `Project project` kürzen, und die einzelnen Attribute mit den Gettern abfragen.